metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -51,13 +75,16 @@ final class Doctor(
 
   /** Checks if there are any potential problems and if any, notifies the user. */
   def check(): Unit = {
-    problemSummary match {
+    val summary = problemSummary
```

Comment:
is this useful?

Reply:
Just to avoid double analysis that runs through all workspace build targets (which could be large), but I admit this is probably not a bottle-neck

Reply:
Ah! I didn't realize `problemSummary` was a method